utils.go

```diff
@@ -43,3 +45,51 @@ func putBuf(b *bytes.Buffer) {
 	b.Reset()
 	bytesBufferPool.Put(b)
 }
+
+// fieldsASCII is similar to strings.Fields but only allows ASCII whitespaces
+func fieldsASCII(s string) []string {
+	fn := func(r rune) bool {
+		switch r {
+		case '\t', '\n', '\f', '\r', ' ':
+			return true
+		}
+		return false
+	}
+	return strings.FieldsFunc(s, fn)
+}
+
+// parsePSOutput parses the runtime's ps raw output and returns a TopBody
+func parsePSOutput(output []byte) (*TopBody, error) {
+	topBody := &TopBody{}
+
+	lines := strings.Split(string(output), "\n")
+	topBody.Titles = fieldsASCII(lines[0])
+
+	pidIndex := -1
+	for i, name := range topBody.Titles {
```

Comment:
What is this validation for? Does it really matter if Pid is not in the output?

Reply:
Well, from the containerd and, of course, `docker top`, PID is a must to have. But from go-runc, I think it doesn't. I will remove this validation.

Reply:
One purpose is to ensure the struct will contain the PIDs and another is to handle the output when the `m` option is passed.